### DISCUSSION TITLE
AMD / UMD support how to use with requirejs ?

### DISCUSSION THREAD

**Discussion by trystanclarke** [JedWatson/react-select#4605] [Q&A]
I work on a legacy project that uses requirejs, which we are gradually converting to react.  
Do you have an example of how to load the built js files with requirejs please? I didn't see an AMD / UMD file in the dist folder?

**Comment by ebonow** [comment]
Unfortunately, UMD builds have been removed as of version 3
https://github.com/JedWatson/react-select/releases/tag/react-select%403.0.3

**Comment by luanped** [comment]
Is there any reason as to why UMD builds were dropped ? I find a lot of places mentioning this fact, but no reason as to why?
